include/sound/soc-dpcm.h

```diff
@@ -158,10 +158,12 @@ enum snd_soc_dpcm_state
 void snd_soc_dpcm_be_set_state(struct snd_soc_pcm_runtime *be, int stream,
 	enum snd_soc_dpcm_state state);
 
+/* update audio routing between PCMs and any DAI links */
```

Comment:
I would also give an example of the use case in the commit message so that others on the list can follow.

Reply:
@lgirdwood Sorry, the commit message already contains a use case example. Would you like me to replace or extend it or to add another one?

Reply:
As an example, external virtualised ASoC component drivers may need to call these when managing shared DAPM routes that are used by more than one driver (i.e. host kernel driver and guest drivers have a DAPM path from guest PCM to host DAI).

Reply:
Thanks, updated